packages/preview/university-assignment/0.1.0/README.md

```diff
@@ -0,0 +1,153 @@
+# University Assignment Template
+
+A flexible Typst template for university assignments, lab reports, and homework with a clean, professional layout.
+
+## Features
+
+- **Clean, Professional Layout**: Modern design perfect for academic submissions
+- **Customizable Assignment Details**: Support for course info, instructors, due dates, hardware/software used, and more
+- **Automatic Table of Contents**: Generated from your document structure
+- **Styled Elements**: Beautiful code blocks, headings, quotes, and lists
+- **Flexible Configuration**: Choose which details to include based on your assignment type
+
+## Installation
+
+### From Typst Package Registry (when published)
```

Comment:
This README will appear on Universe as is once published, so you should remove the "(when published)" mention here.